app/pb_kits/playbook/pb_distribution_bar/_distribution_bar.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= content_tag(:div,
```

Comment:
Did you consider using a [figure tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure)? It would be a more semantic.

Reply:
I think this is default from the playbook kit generator and thus the pattern for most kits in playbook.